### PR TITLE
Add new CM's Snap camera to stock cameras list

### PR DESCRIPTION
**Pull request by eugenesan**

### CODE REVIEW

File: scripts/inc.installdata.sh [hunk 1]
```diff
@@ -221,6 +221,7 @@ priv-app/Camera'"$REMOVALSUFFIX"'
 priv-app/Camera2'"$REMOVALSUFFIX"'
 app/MotCamera'"$REMOVALSUFFIX"'
 priv-app/MotCamera'"$REMOVALSUFFIX"'
+app/Snap/Snap'"$REMOVALSUFFIX"'
```
Comment (mfonville) [inline scripts/inc.installdata.sh]: are you sure it is twice in a subdirectory?
shouldn't it be just app/Snap and not app/Snap/Snap?

**Comment by eugenesan** [comment]
Fixed redundant subfolder, thanks for pointing out.

**Comment by mfonville** [comment]
:-)

Lgtm now

**Comment by gericom-hummer** [comment]
Is it possible to leave us a choice to bypass stock camera when installing Google cam?

**Pull request merged**